agg-ast/ast/src/definitions.rs

```diff
@@ -532,10 +533,50 @@ pub struct SubqueryLookup {
     pub as_var: String,
 }
 
-#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize, Eq, PartialOrd, Ord)]
 pub struct Namespace {
-    pub db: String,
-    pub coll: String,
+    #[serde(alias = "db")]
```

Comment:
the benefit of aliasing here is that it allows us to read in aggregation pipelines as is ("db", "coll" for lookups) but it also allows us to serialize to "database" "collection" (preserving translation format) and deserialize again